### PR TITLE
add GCP health check nginx config for Tahoe

### PR DESCRIPTION
**Pull request by thraxil**
Tahoe's letsencrypt setup is in the way of the usual GCP load balancer health checking. This just adds a separate virtual host config that will listen for just those health check requests and handle them appropriately.

I've included both the edX `/heartbeat` endpoint, which gets proxied back to the LMS gunicorn process (it was a totally arbitrary choice between LMS and CMS. The LMS is probably the more critical one to ensure is running), which checks the database and mongodb connections.

If that proves to be too much of a load on the server (GCP's health checker hits a couple times per second), there's also a lighter weight `/status/` that we can use that nginx responds to immediately. This seems to be closer to upstream's approach to health checking (though they match based on the user agent rather than path).

### CODE REVIEW

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/health-check.j2 [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# config to pass GCP load balancer health checks along to the
+# /heartbeat endpoint of the LMS and allow lighter weight checks
+# on /status/
+
+upstream health-check-backend {
+  {% for host in nginx_cms_gunicorn_hosts %}
```
Comment (OmarIthawi) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/health-check.j2]: So `nginx_cms_gunicorn_hosts` forces CMS connection to be redirected to the LMS?

**Review by OmarIthawi — APPROVED** [review]
One question though.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/health-check.j2 (hunk 1 above)
Comment (thraxil) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/health-check.j2]: Just a typo. Fixed.

**Review by OmarIthawi — APPROVED** [review]
LGTM now!

**Pull request merged**